app/src/main/res/values-ca/strings.xml

```diff
@@ -0,0 +1,47 @@
+<!--
+  ~ Copyright 2019 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <string name="menu_filter_by_grow_zone">Filtrar per zona de creixement</string>
+    <string name="plant_detail_image_content_description">Imatge de la planta</string>
+    <string name="my_garden_title">El meu jardí</string>
+    <string name="plant_list_title">Llistat de plantes</string>
+    <string name="available">Plantes Disponibles</string>
+    <string name="plant_details_title">Detalls de la planta</string>
+    <string name="add_plant">Afegir planta</string>
+    <string name="added_plant_to_garden">Planta afegida al jardí</string>
+    <string name="garden_empty">No hi ha plantes al teu jardí</string>
+    <string name="plant_date_header">Plantada</string>
+    <string name="watered_date_header">Regada per última vegada</string>
+    <string name="menu_item_share_plant">Compartir</string>
+    <string name="share_text_plant">Revisa la planta %s a l\'aplicació Android Sunflower</string>
+
+    <!-- String plurals and their related prefix / suffix strings -->
+    <string name="watering_needs_prefix">Necessita ser regada</string>
```

Comment:
In Catalan, it's better to use "caldre" instead of "necessitar". I would suggest:
```suggestion
    <string name="watering_needs_prefix">Cal regar-la</string>
```